apps/react-storybook/stories/chat/Chat.stories.tsx

```diff
@@ -30,6 +30,10 @@ const commonArgs: ChatTypes.Properties = {
     activeStateEnabled: true,
     hoverStateEnabled: true,
     focusStateEnabled: true,
+    showDayHeaders: true,
```

Comment:
After adding options to commonArgs, we will see these options in the select panel for all stories. 
<img width="382" alt="image" src="https://github.com/user-attachments/assets/25551b44-ba3c-44bc-8d0e-456981b9e87f">


To make it work, we need to pass the options to the component through all stories.

We have several options:
- Demonstrate the options in all stories.
- Add a separate story to demonstrate how these options work there
- Demonstrate them only in the overview demo.

Option two is preferable to me

Reply:
I have created a separate story (Customization) in which all the other options are hidden on the controls tab.
f6007c1b13d32d0c3698e6703fbb27445393f7e6